src/EA.Weee.Web/Areas/Aatf/ViewModels/FilterViewModel.cs

```diff
@@ -1,5 +1,6 @@
 ﻿namespace EA.Weee.Web.Areas.Aatf.ViewModels
 {
+    using System;
```

Comment:
I think this is unused

Reply:
Will tidy this up after Megs changes come in...